packages/cli/CHANGELOG.md

```diff
@@ -73,22 +76,6 @@ LTS Breaking: Removed the following previously deprecated items: [#1981](https:/
 
 - Major: First major version bump for V3
 
-## `7.23.3`
```

Comment:
I do not think that we will want recent V2 changelog entries to be deleted. They should remain.